src/seismometer/_api.py

```diff
@@ -1058,7 +1243,157 @@ def show_cohort_summaries(by_target: bool = False, by_score: bool = False):
     """
     dfs = _get_cohort_summary_dataframes(by_target, by_score)
 
-    display(template.render_cohort_summary_template(dfs))
+    return template.render_cohort_summary_template(dfs)
+
+
+# endregion
+
+# region Exploration Widgets
+
+
+@export
+class ExploreModelEvaluation(ExplorationModelSubgroupEvaluationWidget):
+    """
+    A widget for exploring the model performance of a cohort.
+    """
+
+    def __init__(self):
+        """
+        Exploration widget for model evaluation.
+
+        This includes the ROC, recall vs predicted condition prevalence, calibration,
+        PPV vs sensitivity, sensitivity/specificity/ppv, and a histogram.
+        """
+        from seismometer._api import plot_model_evaluation
+
+        super().__init__("Model Performance", plot_model_evaluation)
```

Comment:
Question: Perhaps we should have a sidebar on the goal for how these are interacted with (I'm fine not holding this PR up over this as I think a couple followups are already expected), but with the simplicity here, I think there are options around removal of this child class completely and doing something more direct like:  
`explore_model_evaluation = ExplorationModelSubgroupEvaluationWidget("Model Performance", plot_model_evaluation)`

And at that point, we could still "act" like a function by adding a `__call__: return self` if we don't like the look of jupyter cells that are recalling.  
I suppose the real decision-point is that a single instance means multiple cells can become linked, for better or worse.

Reply:
neverminding: the "linking" is a problem.  the class is a good way to avoid it.